nservicebus/testing/index_content_testing_[6,).partial.md

```diff
@@ -1,15 +1,37 @@
 ## Testing a handler
 
+Testing a message handler is done using the `TestableMessageHandlerContext` class provided by the `NServiceBus.Testing` package. This class implements `IMessageHandlerContext` and can be passed to the handler under test. After the handler is executed, the `TestableMessageHandlerContext` can be interrogated to assert that various actions (sending a message, publishing an event, etc.) occurred in the handler as expected.
+
+| Property | Description |
+|---------------------|----------------------------------------------------------------------------------|
+| `SentMessages` | A list of all messages sent by `context.Send()`. |
+| `PublishedMessages` | A list of all messages published by `context.Publish()`. |
+| `RepliedMessages` | A list of all messages sent using `context.Reply()`. |
+| `TimeoutMessages` | A list of all messages sent with a saga timeout header. |
```

Comment:
why not just state using the RequestTimeout API instead of saga timeout header?

Reply:
ok I understand that, it doesn't seem to be a good reason not to mention the RequestTimeout API tough (especially as the other collections here are also linked to related APIs)

Reply:
I'm with Tim, mentioning the RequestTimeout API is more future proof IMO since me and @SzymonPobiega have an evil plan to move the responsibility for timeouts to the persisters instead. Ie no message would be actually sent